### ISSUE TITLE
Rule to forbid `yield` in fixtures without teardown

### ISSUE THREAD

**Issue by sobolevn** [m-burst/flake8-pytest-style#57]
# Rule request

## Description

We only need fixtures like this one:

```python
@pytest.fixture
def some_fixture():
    yield 1
    print('Teardown')
```

When there's actually a teardown process. I sometimes see a code like this:

```python
@pytest.fixture
def some_fixture():
    yield 1
```

Or:

```python
@pytest.fixture
def some_fixture():
    print('Setup')
    yield 1
```

So, basically I suggest to forbid that `yield` statement is the last one in a function's body.

## Rationale

It does not make sence. And semantically incorrect. Use `return 1` instead.
